helm/minio/values.yaml

```diff
@@ -315,6 +315,10 @@ environment:
   ## MINIO_SUBNET_LICENSE: "License key obtained from https://subnet.min.io"
   ## MINIO_BROWSER: "off"
 
+## The name of a secret in the same kubernetes namespace which contain values to be added to the environment
+## This can be useful for LDAP password, etc
+# envFromSecret: minio-extraenv
+
```

Comment:
You can actually mount the entire secret directly as `config.env` all you need to set is `MINIO_CONFIG_ENV_FILE=path/to/env` - MinIO will load the entries in this and set the relevant environment values. 

Expected format is like 

```
ENV_NAME1=value
ENV_NAME2=value
```

This allows overriding all values. It is preferred to have this change over loading from `envFrom`

Reply:
Thanks, that would also do what I intend to do here, although I don't see from the templates how would it be possible to mount an additional secret right now. I could change this PR in order to do that if that's ok for you.

Reply:
yes that is better way to do it @Sea-you